jcl/source/windows/JclRegistry.pas

```diff
@@ -398,7 +402,7 @@ implementation
   Winapi.AccCtrl,
   {$ELSE ~HAS_UNITSCOPE}
   SysUtils,
-  AccCtrl,
+  {$IFDEF DELPHILANGUAGE}AccCtrl,{$ENDIF ~DELPHILANGUAGE}
```

Comment:
Delphi does not define `DELPHILANGUAGE` so basically, you removed `AccCtrl` for all Delphi versions. And the `~` in the `ENDIF` is specifically for when using `IFNDEF` which is not the case here.